README.md

```diff
@@ -57,19 +91,27 @@ It is highly recommended that you use a tool like [`nvm`](https://github.com/nvm
     $ npm run prepare
     $ npm start
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and opens up a browser window. Most changes are 
```

Comment:
Duplicate content here. "Running this project" and "Local Development"  have the same steps.